tools/dockerfile/test/python_alpine_x64/Dockerfile

```diff
@@ -32,14 +34,21 @@ RUN apk update && apk add \
   strace \
   python-dev \
   py-pip \
+  python3-dev \
   unzip \
   wget \
   zip
 
-# Install Python packages from PyPI
-RUN pip install --upgrade pip==10.0.1
-RUN pip install virtualenv
-RUN pip install futures==2.2.0 enum34==1.0.4 protobuf==3.5.0.post1 six==1.10.0
+# Install Python 2 for tooling code.
+RUN pip install --upgrade pip==10.0.1 && \
+	pip install virtualenv futures==2.2.0 enum34==1.0.4 \
```

Comment:
Do we need `futures` and `enum34`?

Reply:
The Python 2 install here is not for running `grpcio`. It is for running the Python 2-only scripts in the `tools/` directory.